### ISSUE TITLE
What minimum amount of GPU is needed to finetune pretrained ? 8gb GPU is enough? or i need 12 gb GPU?

### ISSUE THREAD

**Issue by siddharth2022** [deepmind/kinetics-i3d#65]

**Comment by joaoluiscarreira** [comment]
12GB or 16GB will make things faster (and better for batch norm) but 8 GB
may be enough to finetune, you'll just need to adjust the batch size
appropriately.

Best,

Joao

**Comment by siddharth2022** [comment]
Thanks for the help @joaoluiscarreira 
can u help me to find the optical flow of new dataset for fine tune it.

**Comment by YuLengChuanJiang** [comment]
@joaoluiscarreira Hello, Joao. I want to know how much storage space is required to extract RGB and flow frames from all Kinetics400 videos.

Best
